### ISSUE TITLE
Semantic Versioning

### ISSUE THREAD

**Issue by nrdxp** [input-output-hk/bitte#126]
Originally, I wanted to start the same versioning scheme here I had initiated for bitte cli, which was the semver semantic. At the time, it was argued that calendar version may be more appropriate, and that's what was initiated by others.

However, now that we are maintaining release branches for patches and hot-fixes, it really seems that semantic versioning would be more appropriate. Adding additional commits to a date versioning release kind of destroys the meaning of the version, and the path release number `x.x.[0-9]` seems more appropriate for that kind of thing.

**Comment by jonringer** [comment]
Agreed. CalVer is more oriented around "packages with a static interface, but changing content". Bitte is under heavy development right now, and the API is changing greatly.
